dataplanes/vpp/pkg/nsmutils/nsm-utils.go

```diff
@@ -69,9 +68,10 @@ func ValidateParameters(parameters map[string]string, keyList Keys) error {
 
 // Keys validator functions, for each new Keys there should be a validator function.
 func Namespace(value string) error {
-	if !strings.HasPrefix(value, "pid:") {
-		return fmt.Errorf("malformed namespace %s, must start with \"pid:\" following by the process id of a container", value)
-	}
+	// This is not true anymore
+	// if !strings.HasPrefix(value, "pid:") {
+	// 	return fmt.Errorf("malformed namespace %s, must start with \"pid:\" following by the process id of a container", value)
+	// }
```

Comment:
We should probably either eliminate this function entirely or expand the validator to the new circumstances.

Reply:
@edwarnicke Done... current validation mechanism seems to be limited in terms of expressiveness, and kernel-kernel, for example, performs additional validation steps, so I'd better remove this validation in the future in favour of hard-coded verification of parameters